test/functional/api/server_requests_spec.lua

```diff
@@ -32,6 +32,20 @@ describe('server -> client', function()
     end)
   end)
 
+  describe('NULL-initialized string handling', function()
```

Comment:
I think here a comment is needed about where you saw `NULL`. I could not guess this without an issue from the test code only.